src/css/controls/imports/slider.less

```diff
@@ -181,6 +181,10 @@
         left: 0;
     }
 
+    .jw-breakpoint-0.jw-flag-touch &::before {
+        height: 34px;
+    }
+
```

Comment:
I slightly decreased the 'fat-finger touch area' on the time slider from 44px to 34px